fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -698,6 +712,9 @@ def cli():
     parser.add_argument(
         '-a', '--analyze-only', required=False, action='store_true',
         help='If present, will perform analysis, but do no new queries.')
+    parser.add_argument(
+        '-A', '--analytical-tables-only', required=False, action='store_true',
```

Comment:
@rohaher I created a temporary flag for some analytical tables I'm making for Davera. Two of our reports generation functions are pretty slow. Eventually will want to optimize performance. I think that falls better under my scope than yours.

Reply:
I changes this flag to `--plots-only`